libs/wingsdk/src/utils/aws/topic.name.ts

```diff
@@ -0,0 +1,29 @@
+import { Construct } from "constructs";
+import { NameOptions } from "../name-options";
+
+/**
+ * AWS rules for Topic Names
+ *
+ * Topic names are limited to 256 characters.
+ * You can use alphanumeric characters, hyphens (-) and underscores (_).
+ */
+export class TopicName {
```

Comment:
Lots of duplicated code and boilerplate. Feels like all of these can basically be folded into one or two utility methods that are not resource specific and then we can call them from each resource with the specific constraints.

WDYT?

Reply:
You're right, I will make some changes to avoid this